### ISSUE TITLE
Capture response to file for a specific response

### ISSUE THREAD

**Issue by jcamiel** [Orange-OpenSource/hurl#1326]
### Discussed in https://github.com/Orange-OpenSource/hurl/discussions/1323

<div type='discussions-op-text'>

<sup>Originally posted by **tpluscode** March  2, 2023</sup>
I have a resource like this: 

```json
{
  "id": "foobar",
  "images": {
    "small": "...url...",
    "medium": "...url...",
    "large": "...url..."
  }
}
```

It would be nice to "capture" entire responses as files within chained requests. I would capture the image URLs and follow the first request with 3 more to get the images

```
GET /resource

HTTP/* 200
[Captures]
id: jsonpath "$.id"
image-small: jsonpath "$.images.small"
image-medium: jsonpath "$.images.medium"
image-large: jsonpath "$.images.large"

GET {{image-small}}

HTTP/* 200
[Captures]
save-as: {{id}}/small.jpg

GET {{image-medium}}

HTTP/* 200
[Captures]
save-as: {{id}}/medium.jpg

GET {{image-large}}

HTTP/* 200
[Captures]
save-as: {{id}}/large.jpg
```</div>

**Comment by jcamiel** [comment]
Hi @tpluscode @RoberGauld @moono the `--output` per request option have landed on `master`:

```hurl
GET https://foo.com
[Options]
output: /tmp/out.bin
HTTP 200

GET https://bar.com
```

If you know how to build a Rust program, I would be graceful to get feedback on the feature. In any case, the option will be available in the next Hurl release.

**Comment by jcamiel** [comment]
Fixes #2184
